src/dialogs/settingsdialog.h

```diff
@@ -48,6 +48,7 @@ Q_OBJECT
         ToolbarPage,
         DebugOptionPage,
         EditorPage,
+        PanelsPage,
```

Comment:
`PanelsPage` needs to be behind `GitPage` and your panel needs to have a `whatsThis` of 16 instead of 15 (because  `GitPage` already has 15)